components/style/values/specified/transform.rs

```diff
@@ -564,3 +565,31 @@ impl Parse for Translate {
         Ok(GenericTranslate::TranslateX(tx))
     }
 }
+
+/// A specified CSS `scale`
+pub type Scale = GenericScale<Number>;
+
+impl Parse for Scale {
+    fn parse<'i, 't>(
+        context: &ParserContext,
+        input: &mut Parser<'i, 't>
+    ) -> Result<Self, ParseError<'i>> {
+        if input.try(|i| i.expect_ident_matching("none")).is_ok() {
+            return Ok(GenericScale::None);
+        }
+
+        let sx = Number::parse(context, input)?;
+        if let Some(sy) = input.try(|i| Number::parse(context, i)).ok() {
```

Comment:
nit: I pointed out on the original review but apparently only some of them were fixed. There's no need to do `.ok()` here, you can just do:

```rust
if let Ok(sy) = input.try(|i| Number::parse(context, i)) {
```

And ditto below.